middleware/profiler.go

```diff
@@ -29,15 +29,11 @@ func Profiler() http.Handler {
 		http.Redirect(w, r, r.RequestURI+"/", 301)
 	})
 
-	r.HandleFunc("/pprof/", pprof.Index)
+	r.HandleFunc("/pprof/*", pprof.Index)
 	r.HandleFunc("/pprof/cmdline", pprof.Cmdline)
 	r.HandleFunc("/pprof/profile", pprof.Profile)
 	r.HandleFunc("/pprof/symbol", pprof.Symbol)
 	r.HandleFunc("/pprof/trace", pprof.Trace)
-	r.Handle("/pprof/block", pprof.Handler("block"))
-	r.Handle("/pprof/heap", pprof.Handler("heap"))
-	r.Handle("/pprof/goroutine", pprof.Handler("goroutine"))
-	r.Handle("/pprof/threadcreate", pprof.Handler("threadcreate"))
```

Comment:
This change would hide all these endpoints from [docgen](https://github.com/go-chi/docgen). Can we instead be more explicit and add the missing handlers?

Reply:
The problem is that these are dynamic, you can add a new one using [pprof.NewProfile](https://golang.org/pkg/runtime/pprof/#NewProfile).